resources/build/builder-basic.inc.sh

```diff
@@ -115,6 +116,16 @@ function _builder_basic_find_version() {
     KEYMAN_VERSION_WITH_TAG="$KEYMAN_VERSION$KEYMAN_VERSION_TAG"
     KEYMAN_VERSION_GIT_TAG="release@$KEYMAN_VERSION_WITH_TAG"
 
+    # For local builds, and for PR builds, we will use a filename including the tag
+    # but for release builds, we omit the tag for now. In the future, we may include
+    # the tag but that involves changing a number of other websites as dealing with
+    # the rename there too.
+    if builder_is_ci_release_build; then
+      KEYMAN_VERSION_FOR_PR_FILENAME="${KEYMAN_VERSION}"
```

Comment:
`KEYMAN_VERSION_FOR_PR_FILENAME` is a bit misleading since it's also used for non-PR builds. What about just `KEYMAN_VERSION_FILENAME`?

Reply:
Hmm, could do, may make this a follow-up PR since I have two following PRs...

Reply:
See #14720.